Corona-Warn-App/src/deviceForTesters/java/de/rki/coronawarnapp/test/debugoptions/ui/DebugOptionsFragmentViewModel.kt

```diff
@@ -23,7 +23,18 @@ class DebugOptionsFragmentViewModel @AssistedInject constructor(
     val environmentState = environmentStateFlow.asLiveData(context = dispatcherProvider.Default)
     val environmentStateChange = SingleLiveEvent<EnvironmentState>()
 
-    fun selectEnvironmentTytpe(type: String) {
+    fun clearLaunchEnvironment() {
+        envSetup.launchEnvironment = null
+        envSetup.toEnvironmentState()
+            .copy(isOverwritten = false)
+            .let {
+                environmentStateFlow.value = it
```

Comment:
Maybe also refresh view (hide clear button, show toggleGroup) when launch environment is cleared

Reply:
test menu is fixed